source/data/introduction_to_irods.rst

```diff
@@ -19,14 +19,8 @@ iRODS gives users four core competencies:
     iRODS servers can execute event-triggered background process (rules) to execute defined actions when a particular system activity happens. iRODS event triggers are called Policy Enforcement Points (PEPs). The combination of PEPs and rules allows administrators and users to create powerful, customized workflows for automating procedures that help to save time and prevent human error.
 
 - **Secure Collaboration**
-    A facility to share data between users and user groups in a secure way.
+    iRODS provides facilities to share data between users and user groups in a secure way. iRODS has a permissions system similar to Unix file system permissions. Other facilities offered are 'Tickets' and 'Federation', but these are not actively used on the Tier-1 Data Platform.
```

Comment:
I suppose a better usage is 'permission model' in place of 'permission system' - how I have seen in the usage of irods people.